script/Arcueid/Day 6/06_00_ARC06_2_2_123.txt

```diff
@@ -1,93 +1,105 @@
 [1e7246c9bf60762b5c189e482667f61b70bf135b]{
 -- Page 249, Offset 14512.
 -- 「なーんて、ぜーんぶ見当違いの話でーす！
-"Nope, all those answers are completely wrong!
+"Just kidding, all of those answers are completely wrong!
 }
 [814396974fdb96bf0a7c9b956c9b54695c9e060c]{
 -- Page 249, Offset 14513.
 -- 人為的才能と比べる事さえおこがましい！
-It would be absurd to even compare them against such unnatural talent!
+It would be absurd to even compare her talents to those crafted by the hands of men!
 }
 [8cfd6c45801084c8ab7bbae6a7b8e5a67efdd586]{
 -- Page 249, Offset 14514.
 -- 当主チャンはぁ、万年に一人の怪物だもぉん！」
-Our precious head of the family is a once-in-a-lifetime monster after all!"
+// Once in a lifetime a little small, given the timescales
+Our precious little head of the family is the sort of monster that only shows up once every ten thousand years, after all!"
 }
 [2c29fde5261d2ea92b6fc1a8265700465d8b5c66]{
 -- Page 250, Offset 14515.
 -- ケラケラと、心底から楽しそうに笑う阿良句博士。
-With a cackling sound, Dr. Arach laughs happily from the bottom of her heart.
+Professor Arach lets loose a hearty cackle. It appears she's thoroughly enjoying herself.
 }
 [ceab79a85c2fcec3fa10f619f6dd69ca1e5597a0]{
 -- Page 250, Offset 14516.
 -- ……その態度に、知らず、俺は拳を握りしめていた。
-...I unknowingly clench my fists toward her attitude.
+...I unconsciously clench my fist in response to her attitude.
 }
 [f52ea14409187d9814b3bfabaeff00a426cfd4e0]{
 -- Page 251, Offset 14517.
 -- 「あっらぁ、妹をけなされて怒っちゃったぁ？　アタシとしては最上級の愛情をこめたつもりだったんだけどぉ」
-"Oh my, are you mad because I spoke ill of your sister? But that was my way of showing the highest degree of love."
+"Oh my, are you mad because I spoke ill of your sister?
+ I was only trying to express the extent of my heartfelt affection for her you know."
 }
 [2b582988feaff1148178d9e34df965a300a06612]{
 -- Page 251, Offset 14518.
 -- 「そうですか。だとしても、今の言い方はやめてください」
-"Is that so. Even if that's the case, please don't talk like that ever again."
+"Is that so. Even if that's the case, please do not speak of her like that again."
 }
 [ce4fbf08aadb03d1eb4c6ff7255c4589fa4cb2b6]{
 -- Page 252, Offset 14519.
 -- ……怪物、という喩えはやめてほしい。
-...I don't want her to be described as a monster.
+...I don't want anyone to liken her to a monster.
 }
 [2fd169f28eec54c4eeb72426bb5b937366a7474e]{
 -- Page 252, Offset 14520.
 -- 吸血鬼という本物を知った以上、その単語を人間に使うのは許容できない。
-Now that I know the truth about vampires, I can't tolerate it when humans use that word.
+Now that I know vampires are real, I can't abide anyone using that term to refer to a human.
 }
 [7debfbcd55f030e1cc3ea9058db640cb222166b2]{
 -- Page 252, Offset 14521.
 -- ましてや秋葉に向けるなんて、二度と。
-And against Akiha nonetheless. Never again.
+And to refer to Akiha, at that. I won't allow it.
```

Comment:
Much less directed at Akiha

Reply:
Hm, I kinda prefer the current pattern of refer to human -> specialization -> refer to Akiha. 
> I can't abide anyone using that term to refer to a human.
> Much less directed at Akiha.
just doesn't read as 'connected' to me.